n26/core/templates/n26/includes/equip_owned_line.html

```
@@ -1,89 +1,80 @@
-                {% include "n26/includes/equip_choices.html" with choices=row.buy.choices %}
-            {% endif %}
-            {% if row.buy.options %}
-                {% include "n26/includes/equip_options.html" with row=row.buy %}
-            {% endif %}
-        </div>
-    </c-slot>
-</c-n26.collection-picker.item>
+        </c-slot>
+    </c-n26.collection-picker.item>
+{% else %}
+    <c-n26.collection-picker.item name="{{ row.name }}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)